src/app/public/modules/icon/icon-stack.component.ts

```diff
@@ -0,0 +1,36 @@
+import {
+  Component,
+  Input
+} from '@angular/core';
+
+import {
+  SkyIconStackItem
+} from './icon-stack-item';
+
+@Component({
+  selector: 'sky-icon-stack',
+  templateUrl: './icon-stack.component.html',
+  styleUrls: ['./icon-stack.component.scss']
+})
+export class SkyIconStackComponent {
+
+  /**
+   * Specifies the size of the icon using
+   * [Font Awesome sizes](https://fontawesome.com/how-to-use/on-the-web/styling/sizing-icons).
+   */
```

Comment:
Thanks for adding jsDoc comments. Should we list a `@default` for the `size` property?

Reply:
This is modeled off the icon component property of the same name, and that property doesn't have a default. If the value isn't set, we don't add a size CSS class to the rendered HTML. So there's not really a default value, but default behavior when there's no value present. Given that, do you think it still needs a default specified?

Reply:
Got it. I see the pattern now from the icon docs. If that worked there, then its fine here. Let's move on.